pkg/data/mobileService.go

```diff
@@ -55,6 +55,26 @@ func (ksc keycloakSecretConvertor) Convert(s v1.Secret) (*mobile.ServiceConfig,
 	}, nil
 }
 
+type syncSecretConvertor struct{}
+
+//Convert a kubernetes Sync Server secret into a keycloak mobile.ServiceConfig
+func (scc syncSecretConvertor) Convert(s v1.Secret) (*mobile.ServiceConfig, error) {
+	sc := &mobile.SyncConfig{
+		URI: string(s.Data["uri"]),
+	}
+
+	if threeScaleExists, ok := s.Labels["3scale"]; ok && threeScaleExists == "true" {
+		sc.Headers.AppID = string(s.Data["apicast_app_id"])
```

Comment:
I think we should check for the existence of these before deciding to change the config. These are added by the bind apb and so may not be immediately present. If they are not present yet, I think we should return the "old" config as I think this will provide more consistent behaviour